Assignment3Loops/src/assignment3loops/Assignment3Loops.java

```diff
@@ -0,0 +1,33 @@
+package assignment3loops;
+import java.util.Scanner;
+
+public class Assignment3Loops {
+    public static void main(String[] args) {
+        //Keyboard input
+        Scanner keyboard = new Scanner(System.in);
+        int cont;//do-while variable
+        
+        print("Welcome to Steve's word Mixer-upper!\n\n");
+       do{
+        print("Please Enter A word you would "
+                + "Like to mix up: ");        
+        String word = keyboard.nextLine();//User Input
+                print("\nYour word reversed is: ");
+                for(int i = word.length()-1; i >= 0; i--){
+                    System.out.print(word.charAt(i));  
```

Comment:
Because in the method I have it printing out Strings, not chars. It was not compatible. I figured just to leave it as such, instead of turning it to a char, and typing out the whole line for the strings.